tests/test_config.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from module import ConfigParser
+import os
+
+
+class TestConfig:
+
+    def setUp(self):
+        ConfigParser.pypath = ".."
+        self.config = ConfigParser.ConfigParser()
+
+    def teardown(self):
+        os.unlink('pyload.conf')
+        os.unlink('plugin.conf')
+
+    def test_read(self):
+        assert self.config.get('webinterface', 'template') == 'modern'
+        assert self.config.get('remote', 'port') == 7227
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)